lib/B/Deparse.pm

```diff
@@ -6762,7 +6762,7 @@ sub pp_pushdefer {
 sub builtin1 {
     my $self = shift;
     my ($op, $cx, $name) = @_;
-    my $arg = $self->deparse($op->first);
+    my $arg = $self->deparse($op->first, 6);
```

Comment:
What is the rationale for `6` as the second argument in that method call?

Reply:
This is a sub call, and we're deparsing the argument to the sub, so I looked at how pp_entersub (which handles a normal sub call) deparses arguments:
```
    if ($listargs) {
	$args = join(", ", map(
		    ($_->flags & OPf_WANT) == OPf_WANT_SCALAR
		 && !retscalar($_)
			? $self->maybe_parens_unop('scalar', $_, 6)
			: $self->deparse($_, 6),
		    @exprs
		));
    }
```
which uses a precedence value of 6.